pyworkflow/mapper/sqlite.py

```diff
@@ -764,8 +765,24 @@ def insert(self, obj):
             self.doCreateTables = False
         """Insert a new object into the system, the id will be set"""
         self.db.insertObject(obj.getObjId(), obj.isEnabled(), obj.getObjLabel(), obj.getObjComment(), 
-                             *obj.getObjDict().values())
-        
+                             *self._getValuesFromObject(obj).values())
+
+    def getAttributes2Store(self, item):
```

Comment:
Please use getAttributesToStore instead of getAttributes2Store...the latter does is not consistent with other existing names